DuggaSys/diagram.php

```diff
@@ -464,14 +464,14 @@
             <div>
                 <fieldset>
                     <legend>Zoom</legend>
-                    <input class="zoomButtons" type="button" value="Zoom in" onclick='zoomin();' />
-                    <input class="zoomButtons" type="button" value="Zoom out" onclick='zoomout();' />									
+                    <input class="paneButtons" type="button" value="Zoom in" onclick='zoomin();' />
+                    <input class="paneButtons" type="button" value="Zoom out" onclick='zoomout();' />									
                 </fieldset>
 
-                <input type="button" value="Mouse Selection" onclick='setMouseMode(0);' />
-                <input type="button" value="Entity" onclick='setMouseMode(1);' />
-                <input type="button" value="Relation" onclick='setMouseMode(2);' />
-                <input type="button" value="Attribute" onclick='setMouseMode(3);' />
+                <input class="paneButtons" value="Mouse Selection" onclick='setMouseMode(0);' />
+                <input class="paneButtons" value="Entity" onclick='setMouseMode(1);' />
+                <input class="paneButtons" value="Relation" onclick='setMouseMode(2);' />
+                <input class="paneButtons" value="Attribute" onclick='setMouseMode(3);' />
```

Comment:
This change removes the input-type "button" that makes the input go to the default type value of "text". Therefore it is possible to edit the field.

Reply:
Correct, added button types.